### PR TITLE
Add hidden context to Search nav

### LINKED ISSUE #67: Add hidden context to Search nav
To help make clear that the "Search" item in the main nav is a navigation menu, add hidden but accessible text to the nav: 

Reads: "Search navigation menu"
Displays: "Search" (no change from current"
Code: ``<a class="no-underline" href="/search">Search <span class="sr">navigation menu</span></a>``

(actual class for navigation should be whatever we are currently using to hide text meant for screen readers)

### PR DESCRIPTION
**Pull request by PBruk**
Tagging @frrrances for code-review: resolves #67.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -498,10 +498,12 @@ img.wp-smiley,
 
 /* Assistive text */
 .assistive-text,
-.site .screen-reader-text {
+.site .screen-reader-text,
```
Comment (frrrances) [inline style.css]: why the double class? and can we change this class to ".sr" for screenreader?

File: style.css [hunk 2]
```diff
@@ -497,11 +497,11 @@ img.wp-smiley,
 /* Body, links, basics */
 
 /* Assistive text */
-.assistive-text,
-.site .screen-reader-text {
+.assistive-text {
```
Comment (frrrances) [inline style.css]: Did you mean for this class to not match the html?

**Comment by PBruk** [comment]
IIRC, line 501 was created when I added .screen-reader-text class. The reason I prefixed .site was because the styling didn't work, otherwise. I removed it now, since I am removing that class from the templates I added it to but am leaving .assistive-text because I don't if that still exists somewhere, so I'd rather not break anything.

**Comment by frrrances** [comment]
gotcha - makes sense! thanks for the extra info.

**Comment by frrrances** [comment]
👍

**Pull request merged**